Formula/lasso.rb

```diff
@@ -21,17 +21,19 @@ class Lasso < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "python@3.10" => :build
-  depends_on "six" => :build
   depends_on "glib"
+  depends_on "libxml2"
   depends_on "libxmlsec1"
   depends_on "openssl@1.1"
 
-  uses_from_macos "libxml2"
+  uses_from_macos "python" => :build
+
+  on_linux do
+    depends_on "six" => :build # macOS Python has `six` installed.
```

Comment:
Should we make it `uses_from_macos "six" => :build` then?

Reply:
That only works with macOS Python though, so I wouldn't say it works like other `uses_from_macos` formulae.

Reply:
Right.
Then maybe we should consider bundling `six` with python formulae (to be inlined with system python)?

Reply:
Not opposed to that, but that's a pretty big project that can be done separate from this PR.